pkg/webhook/clusterpropagationpolicy/mutating.go

```diff
@@ -51,9 +51,8 @@ func (a *MutatingAdmission) Handle(_ context.Context, req admission.Request) adm
 		return admission.Errored(http.StatusBadRequest, fmt.Errorf("ClusterPropagationPolicy's name should be no more than %d characters", validation.LabelValueMaxLength))
 	}
 
-	addedResourceSelectors := helper.GetFollowedResourceSelectorsWhenMatchServiceImport(policy.Spec.ResourceSelectors)
-	if addedResourceSelectors != nil {
-		policy.Spec.ResourceSelectors = append(policy.Spec.ResourceSelectors, addedResourceSelectors...)
+	if helper.ContainsServiceImport(policy.Spec.ResourceSelectors) {
+		policy.Spec.PropagateDeps = true
```

Comment:
I wonder who is creating the policy. If it is created by an end user, what if there are some other resources in the resource selector that are not intended to enable `PropagateDeps`?

Just a question, otherwise LGTM.

Reply:
> I wonder who is creating the policy. If it is created by an end user, what if there are some other resources in the resource selector that are not intended to enable PropagateDeps?

enable `PropagateDeps` does not have any side effects.

Reply:
ok then. 
Just feel like this behavior should be noted somewhere.